libs/backend/src/scan/cast_vote_records/build_cast_vote_record.ts

```diff
@@ -372,7 +374,10 @@ function buildOriginalSnapshot({
                 (Math.floor(mark.score * 100) / 100).toString(),
               ],
               HasIndication:
-                mark.score >= definiteMarkThreshold
+                getMarkStatus(mark.score, {
+                  marginal: definiteMarkThreshold,
```

Comment:
This is a bit tricky. Maybe omitting `marginal` would be a clearer way to specify that you want to ignore marginal marks?

Alternatively could change this to check `!== MarkStatus.Unmarked`

Reply:
You can now omit `marginal` as you suggested.